pkg/sdn/plugin/egress_network_policy.go

```diff
@@ -79,3 +91,57 @@ func (plugin *OsdnNode) UpdateEgressNetworkPolicyVNID(namespace string, oldVnid,
 		plugin.updateEgressNetworkPolicyRules(newVnid)
 	}
 }
+
+func (plugin *OsdnNode) syncEgressDNSPolicyRules() {
```

Comment:
Okay, I will make proxy and egress network policy register for dns changes and the go routine in egress dns that updates TTL and IP will inform all the registered callers when there is an IP update.